weave

```diff
@@ -1886,14 +1858,19 @@ EOF
         ;;
     launch)
         deprecation_warnings "$@"
+        check_not_running router $CONTAINER_NAME        $BASE_IMAGE
+        check_not_running proxy  $PROXY_CONTAINER_NAME  $BASE_EXEC_IMAGE
+        # Don't check the plugin here, since we start it with `--restart=always`
```

Comment:
This comment seems inappropriate now we start everything with `--restart=always`.  Presumably we should check all three are not running.